cmd/kots/cli/admin-console-generate-manifests.go

```diff
@@ -31,6 +34,26 @@ func AdminGenerateManifestsCmd() *cobra.Command {
 				return errors.Wrap(err, "failed to get namespace")
 			}
 
+			// set defaults for variables requiring cluster context
+			isOpenShift, isGKEAutopilot := false, false
+			migrateToMinioXl, currentMinioImage := false, ""
+			registryConfig := kotsadmtypes.RegistryConfig{
+				OverrideVersion:   v.GetString("kotsadm-tag"),
+				OverrideRegistry:  v.GetString("kotsadm-registry"),
+				OverrideNamespace: v.GetString("kotsadm-namespace"),
+				Username:          v.GetString("registry-username"),
+				Password:          v.GetString("registry-password"),
+			}
+			if clientset, err := k8sutil.GetClientset(); err == nil {
+				isOpenShift, isGKEAutopilot = k8sutil.IsOpenShift(clientset), k8sutil.IsGKEAutopilot(clientset)
+				migrateToMinioXl, currentMinioImage, _ = kotsadm.IsMinioXlMigrationNeeded(clientset, namespace)
+				if newRegistryConfig, err := getRegistryConfig(v, clientset); err == nil {
+					registryConfig = *newRegistryConfig
+				} else {
+					logger.Error(errors.Wrap(err, "failed to get registry config"))
```

Comment:
how about moving the `log := logger.NewCLILogger(cmd.OutOrStdout())` below to be above this and just use the `log` variable that's already initialized?

Reply:
yup, good call